sklearn/decomposition/tests/test_online_lda.py

```diff
@@ -184,7 +184,8 @@ def test_lda_no_component_error():
     rng = np.random.RandomState(0)
     X = rng.randint(4, size=(20, 10))
     lda = LatentDirichletAllocation()
-    regex = r"^no 'components_' attribute"
+    regex = "This LatentDirichletAllocation instance is not fitted yet. " + \
```

Comment:
We do not use `\` to break line most of the time but the parentheses instead.

```python
regex = ("This LatentDirichletAllocation instance is not fitted yet. "
         "Call 'fit' with appropriate arguments before using this method.")

```

Reply:
Fixed